clang/test/CodeGen/ubsan-handler-merge.c

```diff
@@ -0,0 +1,113 @@
+// NOTE: Assertions have mostly been autogenerated by utils/update_cc_test_checks.py UTC_ARGS: --version 5
+// The most important assertion is the attributes at the end of the file, which
+// shows that -ubsan-unique-traps does not attach 'nomerge' to each ubsan handler.
+//
+// RUN: %clang_cc1 -triple x86_64-linux-gnu -emit-llvm -fsanitize=signed-integer-overflow -O3 -mllvm -ubsan-unique-traps %s -o - \
```

Comment:
This test is good, but most important is https://clang.llvm.org/docs/UndefinedBehaviorSanitizer.html#minimal-runtime

Reply:
Done